octane-eclipse-feature/feature.xml

```diff
@@ -14,17 +14,17 @@
    </copyright>
 
    <license url="http://www.apache.org/licenses/LICENSE-2.0">
-      © Copyright 2017-2022 Micro Focus or one of its affiliates.
-Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-     http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
+      © Copyright 2017-2022 Micro Focus or one of its affiliates.
+Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
```

Comment:
ctrl alt L / ctrl alt o?

Reply:
no... but i can't figure it out, texts are the same... probably some whitespace